control-plane-operator/hostedclusterconfigoperator/controllers/resources/ingress/reconcile.go

```diff
@@ -121,10 +124,13 @@ func ReconcileDefaultIngressPassthroughService(service *corev1.Service, defaultN
 		},
 	}
 	service.Spec.Selector = map[string]string{
-		"kubevirt.io": "virt-launcher",
+		"kubevirt.io":                     "virt-launcher",
+		"hypershift.kubevirt.io/infra-id": hcp.Spec.InfraID,
 	}
 	service.Spec.Type = corev1.ServiceTypeClusterIP
 
+	service.Labels["hypershift.kubevirt.io/infra-id"] = hcp.Spec.InfraID
```

Comment:
the string `"hypershift.kubevirt.io/infra-id"` appears in 2 locations. wouldn't it be better to put it in a variable, e.g. `infraIDLabelKey`, as you did in `MachineTemplateSpec` function

Reply:
oh, yeah definitely. I didn't realize i hadn't transitioned to a variable for that. good call.